docs/examples/demoscenario.md

```diff
@@ -0,0 +1,51 @@
+# Using GitHub Connector <!-- omit in toc -->
+
+- [Overview](#overview)
+- [Prerequisites](#prerequisites)
+- [Steps](#steps)
+
+## Overview
+
+This purpose of this guide is to show example usage of GitHub Connector, which allows you to to handle GitHub events. With it's help you will create a lamda in Kyma, which reacts to new issue on connected repository with a short message accessible in lambda's logs. 
```

Comment:
```suggestion
This purpose of this guide is to show example usage of GitHub Connector, which allows you to to handle GitHub events. With its help, you will create a lamda in Kyma, which reacts to new issues created in the connected repository with a short message accessible in lambda's logs. 
```

it's = it is
its = belonging or pertaining to that thing


The sentence that starts with `with its...` is supposed to describe what the user does and achieves in this example, right? I think that it's all about connecting Kyma to your Github repo, then creating a lambda and testing the connection.